WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+protocol ReaderDetailView: class {
+    func render(_ post: ReaderPost)
+    func showError()
+}
+
+class ReaderDetailWebviewViewController: UIViewController, ReaderDetailView {
+    @IBOutlet weak var webView: WKWebView!
+
+    /// The coordinator, responsible for the logic
+    var coordinator: ReaderDetailCoordinator!
```

Comment:
Can this be an optional instead of force unwrapping?

Reply:
So, my reason behind that is that this VC is useless without the coordinator. This is not intended to be used without a coordinator at all.

If that happens for any reason, I'd prefer the app to crash then to display a white screen.

Wdyt?

Reply:
I never prefer the app crashing!!!! we can show an error when guarding for the coordinator 
Where is the coordinator getting initialized when you're calling `controllerWithPostURL` from `presentReaderDetailViewControllerWithURL' in `ReaderStreamViewController` ?

Reply:
Done in 5ac5cb374a957a16b0c271bc42d323e30bbab397